diff --git a/libc/cmake/modules/CheckCompilerFeatures.cmake b/libc/cmake/modules/CheckCompilerFeatures.cmake
index a5ea66a59..1e870b41f 100644
--- a/libc/cmake/modules/CheckCompilerFeatures.cmake
+++ b/libc/cmake/modules/CheckCompilerFeatures.cmake
@@ -102,32 +102,32 @@ foreach(feature IN LISTS ALL_COMPILER_FEATURES)
     )
   endif()
 
-  if(has_feature)
-    list(APPEND AVAILABLE_COMPILER_FEATURES ${feature})
-    if(${feature} STREQUAL "float16")
-      set(LIBC_TYPES_HAS_FLOAT16 TRUE)
-    elseif(${feature} STREQUAL "float16_conversion")
-      add_compile_definitions(__LIBC_USE_FLOAT16_CONVERSION)
-    elseif(${feature} STREQUAL "float128")
-      set(LIBC_TYPES_HAS_FLOAT128 TRUE)
-    elseif(${feature} STREQUAL "fixed_point")
-      set(LIBC_COMPILER_HAS_FIXED_POINT TRUE)
-    elseif(${feature} STREQUAL "cfloat16")
-      set(LIBC_TYPES_HAS_CFLOAT16 TRUE)
-    elseif(${feature} STREQUAL "cfloat128")
-      set(LIBC_TYPES_HAS_CFLOAT128 TRUE)
-    elseif(${feature} STREQUAL "builtin_ceil_floor_rint_trunc")
-      set(LIBC_COMPILER_HAS_BUILTIN_CEIL_FLOOR_RINT_TRUNC TRUE)
-    elseif(${feature} STREQUAL "builtin_fmax_fmin")
-      set(LIBC_COMPILER_HAS_BUILTIN_FMAX_FMIN TRUE)
-    elseif(${feature} STREQUAL "builtin_fmaxf16_fminf16")
-      set(LIBC_COMPILER_HAS_BUILTIN_FMAXF16_FMINF16 TRUE)
-    elseif(${feature} STREQUAL "builtin_round")
-      set(LIBC_COMPILER_HAS_BUILTIN_ROUND TRUE)
-    elseif(${feature} STREQUAL "builtin_roundeven")
-      set(LIBC_COMPILER_HAS_BUILTIN_ROUNDEVEN TRUE)
-    endif()
-  endif()
+  # if(has_feature)
+  #   list(APPEND AVAILABLE_COMPILER_FEATURES ${feature})
+  #   if(${feature} STREQUAL "float16")
+  #     set(LIBC_TYPES_HAS_FLOAT16 TRUE)
+  #   elseif(${feature} STREQUAL "float16_conversion")
+  #     add_compile_definitions(__LIBC_USE_FLOAT16_CONVERSION)
+  #   elseif(${feature} STREQUAL "float128")
+  #     set(LIBC_TYPES_HAS_FLOAT128 TRUE)
+  #   elseif(${feature} STREQUAL "fixed_point")
+  #     set(LIBC_COMPILER_HAS_FIXED_POINT TRUE)
+  #   elseif(${feature} STREQUAL "cfloat16")
+  #     set(LIBC_TYPES_HAS_CFLOAT16 TRUE)
+  #   elseif(${feature} STREQUAL "cfloat128")
+  #     set(LIBC_TYPES_HAS_CFLOAT128 TRUE)
+  #   elseif(${feature} STREQUAL "builtin_ceil_floor_rint_trunc")
+  #     set(LIBC_COMPILER_HAS_BUILTIN_CEIL_FLOOR_RINT_TRUNC TRUE)
+  #   elseif(${feature} STREQUAL "builtin_fmax_fmin")
+  #     set(LIBC_COMPILER_HAS_BUILTIN_FMAX_FMIN TRUE)
+  #   elseif(${feature} STREQUAL "builtin_fmaxf16_fminf16")
+  #     set(LIBC_COMPILER_HAS_BUILTIN_FMAXF16_FMINF16 TRUE)
+  #   elseif(${feature} STREQUAL "builtin_round")
+  #     set(LIBC_COMPILER_HAS_BUILTIN_ROUND TRUE)
+  #   elseif(${feature} STREQUAL "builtin_roundeven")
+  #     set(LIBC_COMPILER_HAS_BUILTIN_ROUNDEVEN TRUE)
+  #   endif()
+  # endif()
 endforeach()
 
 set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
diff --git a/libc/config/baremetal/x86_64/entrypoints.txt b/libc/config/baremetal/x86_64/entrypoints.txt
new file mode 100644
index 000000000..bd90fd823
--- /dev/null
+++ b/libc/config/baremetal/x86_64/entrypoints.txt
@@ -0,0 +1,289 @@
+set(TARGET_LIBC_ENTRYPOINTS
+    # assert.h entrypoints
+    libc.src.assert.__assert_fail
+
+    # compiler entrypoints (no corresponding header)
+    libc.src.compiler.__stack_chk_fail
+
+    # ctype.h entrypoints
+    libc.src.ctype.isalnum
+    libc.src.ctype.isalpha
+    libc.src.ctype.isascii
+    libc.src.ctype.isblank
+    libc.src.ctype.iscntrl
+    libc.src.ctype.isdigit
+    libc.src.ctype.isgraph
+    libc.src.ctype.islower
+    libc.src.ctype.isprint
+    libc.src.ctype.ispunct
+    libc.src.ctype.isspace
+    libc.src.ctype.isupper
+    libc.src.ctype.isxdigit
+    libc.src.ctype.toascii
+    libc.src.ctype.tolower
+    libc.src.ctype.toupper
+
+    # ctype.h entrypoints
+    libc.src.ctype.isalnum_l
+    libc.src.ctype.isalpha_l
+    libc.src.ctype.isblank_l
+    libc.src.ctype.iscntrl_l
+    libc.src.ctype.isdigit_l
+    libc.src.ctype.isgraph_l
+    libc.src.ctype.islower_l
+    libc.src.ctype.isprint_l
+    libc.src.ctype.ispunct_l
+    libc.src.ctype.isspace_l
+    libc.src.ctype.isupper_l
+    libc.src.ctype.isxdigit_l
+    libc.src.ctype.tolower_l
+    libc.src.ctype.toupper_l
+
+    # errno.h entrypoints
+    libc.src.errno.errno
+
+    # locale.h entrypoints
+    libc.src.locale.localeconv
+    libc.src.locale.duplocale
+    libc.src.locale.freelocale
+    libc.src.locale.localeconv
+    libc.src.locale.newlocale
+    libc.src.locale.setlocale
+    libc.src.locale.uselocale
+
+    # setjmp.h entrypoints
+    libc.src.setjmp.longjmp
+    libc.src.setjmp.setjmp
+
+    # string.h entrypoints
+    libc.src.string.memccpy
+    libc.src.string.memchr
+    libc.src.string.memcmp
+    libc.src.string.memcpy
+    libc.src.string.memmem
+    libc.src.string.memmove
+    libc.src.string.mempcpy
+    libc.src.string.memrchr
+    libc.src.string.memset
+    libc.src.string.memset_explicit
+    libc.src.string.stpcpy
+    libc.src.string.stpncpy
+    libc.src.string.strcasestr
+    libc.src.string.strcat
+    libc.src.string.strchr
+    libc.src.string.strchrnul
+    libc.src.string.strcmp
+    libc.src.string.strcoll
+    libc.src.string.strcpy
+    libc.src.string.strcspn
+    libc.src.string.strdup
+    libc.src.string.strerror
+    libc.src.string.strerror_r
+    libc.src.string.strlcat
+    libc.src.string.strlcpy
+    libc.src.string.strlen
+    libc.src.string.strncat
+    libc.src.string.strncmp
+    libc.src.string.strncpy
+    libc.src.string.strndup
+    libc.src.string.strnlen
+    libc.src.string.strpbrk
+    libc.src.string.strrchr
+    libc.src.string.strsep
+    libc.src.string.strspn
+    libc.src.string.strstr
+    libc.src.string.strtok
+    libc.src.string.strtok_r
+    libc.src.string.strxfrm
+
+    # string.h entrypoints
+    libc.src.string.strcoll_l
+    libc.src.string.strxfrm_l
+
+    # strings.h entrypoints
+    libc.src.strings.bcmp
+    libc.src.strings.bcopy
+    libc.src.strings.bzero
+    libc.src.strings.ffs
+    libc.src.strings.ffsl
+    libc.src.strings.ffsll
+    libc.src.strings.index
+    libc.src.strings.rindex
+    libc.src.strings.strcasecmp
+    libc.src.strings.strncasecmp
+
+    # strings.h entrypoints
+    libc.src.strings.strcasecmp_l
+    libc.src.strings.strncasecmp_l
+
+    # inttypes.h entrypoints
+    libc.src.inttypes.imaxabs
+    libc.src.inttypes.imaxdiv
+    libc.src.inttypes.strtoimax
+    libc.src.inttypes.strtoumax
+
+    # stdio.h entrypoints
+    libc.src.stdio.asprintf
+    libc.src.stdio.getchar
+    libc.src.stdio.printf
+    libc.src.stdio.putchar
+    libc.src.stdio.puts
+    libc.src.stdio.remove
+    libc.src.stdio.scanf
+    libc.src.stdio.snprintf
+    libc.src.stdio.sprintf
+    libc.src.stdio.sscanf
+    libc.src.stdio.vasprintf
+    libc.src.stdio.vprintf
+    libc.src.stdio.vscanf
+    libc.src.stdio.vsnprintf
+    libc.src.stdio.vsprintf
+    libc.src.stdio.vsscanf
+
+    # stdbit.h entrypoints
+    libc.src.stdbit.stdc_bit_ceil_uc
+    libc.src.stdbit.stdc_bit_ceil_ui
+    libc.src.stdbit.stdc_bit_ceil_ul
+    libc.src.stdbit.stdc_bit_ceil_ull
+    libc.src.stdbit.stdc_bit_ceil_us
+    libc.src.stdbit.stdc_bit_floor_uc
+    libc.src.stdbit.stdc_bit_floor_ui
+    libc.src.stdbit.stdc_bit_floor_ul
+    libc.src.stdbit.stdc_bit_floor_ull
+    libc.src.stdbit.stdc_bit_floor_us
+    libc.src.stdbit.stdc_bit_width_uc
+    libc.src.stdbit.stdc_bit_width_ui
+    libc.src.stdbit.stdc_bit_width_ul
+    libc.src.stdbit.stdc_bit_width_ull
+    libc.src.stdbit.stdc_bit_width_us
+    libc.src.stdbit.stdc_count_ones_uc
+    libc.src.stdbit.stdc_count_ones_ui
+    libc.src.stdbit.stdc_count_ones_ul
+    libc.src.stdbit.stdc_count_ones_ull
+    libc.src.stdbit.stdc_count_ones_us
+    libc.src.stdbit.stdc_count_zeros_uc
+    libc.src.stdbit.stdc_count_zeros_ui
+    libc.src.stdbit.stdc_count_zeros_ul
+    libc.src.stdbit.stdc_count_zeros_ull
+    libc.src.stdbit.stdc_count_zeros_us
+    libc.src.stdbit.stdc_first_leading_one_uc
+    libc.src.stdbit.stdc_first_leading_one_ui
+    libc.src.stdbit.stdc_first_leading_one_ul
+    libc.src.stdbit.stdc_first_leading_one_ull
+    libc.src.stdbit.stdc_first_leading_one_us
+    libc.src.stdbit.stdc_first_leading_zero_uc
+    libc.src.stdbit.stdc_first_leading_zero_ui
+    libc.src.stdbit.stdc_first_leading_zero_ul
+    libc.src.stdbit.stdc_first_leading_zero_ull
+    libc.src.stdbit.stdc_first_leading_zero_us
+    libc.src.stdbit.stdc_first_trailing_one_uc
+    libc.src.stdbit.stdc_first_trailing_one_ui
+    libc.src.stdbit.stdc_first_trailing_one_ul
+    libc.src.stdbit.stdc_first_trailing_one_ull
+    libc.src.stdbit.stdc_first_trailing_one_us
+    libc.src.stdbit.stdc_first_trailing_zero_uc
+    libc.src.stdbit.stdc_first_trailing_zero_ui
+    libc.src.stdbit.stdc_first_trailing_zero_ul
+    libc.src.stdbit.stdc_first_trailing_zero_ull
+    libc.src.stdbit.stdc_first_trailing_zero_us
+    libc.src.stdbit.stdc_has_single_bit_uc
+    libc.src.stdbit.stdc_has_single_bit_ui
+    libc.src.stdbit.stdc_has_single_bit_ul
+    libc.src.stdbit.stdc_has_single_bit_ull
+    libc.src.stdbit.stdc_has_single_bit_us
+    libc.src.stdbit.stdc_leading_ones_uc
+    libc.src.stdbit.stdc_leading_ones_ui
+    libc.src.stdbit.stdc_leading_ones_ul
+    libc.src.stdbit.stdc_leading_ones_ull
+    libc.src.stdbit.stdc_leading_ones_us
+    libc.src.stdbit.stdc_leading_zeros_uc
+    libc.src.stdbit.stdc_leading_zeros_ui
+    libc.src.stdbit.stdc_leading_zeros_ul
+    libc.src.stdbit.stdc_leading_zeros_ull
+    libc.src.stdbit.stdc_leading_zeros_us
+    libc.src.stdbit.stdc_trailing_ones_uc
+    libc.src.stdbit.stdc_trailing_ones_ui
+    libc.src.stdbit.stdc_trailing_ones_ul
+    libc.src.stdbit.stdc_trailing_ones_ull
+    libc.src.stdbit.stdc_trailing_ones_us
+    libc.src.stdbit.stdc_trailing_zeros_uc
+    libc.src.stdbit.stdc_trailing_zeros_ui
+    libc.src.stdbit.stdc_trailing_zeros_ul
+    libc.src.stdbit.stdc_trailing_zeros_ull
+    libc.src.stdbit.stdc_trailing_zeros_us
+
+    # stdlib.h entrypoints
+    libc.src.stdlib._Exit
+    libc.src.stdlib.a64l
+    libc.src.stdlib.abort
+    libc.src.stdlib.abs
+    libc.src.stdlib.aligned_alloc
+    # libc.src.stdlib.atof
+    libc.src.stdlib.atoi
+    libc.src.stdlib.atol
+    libc.src.stdlib.atoll
+    libc.src.stdlib.bsearch
+    libc.src.stdlib.calloc
+    libc.src.stdlib.div
+    libc.src.stdlib.exit
+    libc.src.stdlib.free
+    libc.src.stdlib.l64a
+    libc.src.stdlib.labs
+    libc.src.stdlib.ldiv
+    libc.src.stdlib.llabs
+    libc.src.stdlib.lldiv
+    libc.src.stdlib.malloc
+    libc.src.stdlib.memalignment
+    libc.src.stdlib.qsort
+    libc.src.stdlib.qsort_r
+    libc.src.stdlib.rand
+    libc.src.stdlib.realloc
+    libc.src.stdlib.srand
+    # libc.src.stdlib.strfromd
+    # libc.src.stdlib.strfromf
+    # libc.src.stdlib.strfroml
+    # libc.src.stdlib.strtod
+    # libc.src.stdlib.strtof
+    libc.src.stdlib.strtol
+    # libc.src.stdlib.strtold
+    libc.src.stdlib.strtoll
+    libc.src.stdlib.strtoul
+    libc.src.stdlib.strtoull
+
+    # stdlib.h entrypoints
+    # libc.src.stdlib.strtod_l
+    # libc.src.stdlib.strtof_l
+    libc.src.stdlib.strtol_l
+    # libc.src.stdlib.strtold_l
+    libc.src.stdlib.strtoll_l
+    libc.src.stdlib.strtoul_l
+    libc.src.stdlib.strtoull_l
+
+    # time.h entrypoints
+    libc.src.time.asctime
+    libc.src.time.asctime_r
+    libc.src.time.clock
+    libc.src.time.ctime
+    libc.src.time.ctime_r
+    # libc.src.time.difftime
+    libc.src.time.gmtime
+    libc.src.time.gmtime_r
+    libc.src.time.mktime
+    libc.src.time.strftime
+    libc.src.time.strftime_l
+    libc.src.time.timespec_get
+
+    # wchar.h entrypoints
+    libc.src.wchar.btowc
+    libc.src.wchar.wcslen
+    libc.src.wchar.wctob
+
+    # internal entrypoints
+    libc.startup.baremetal.init
+    libc.startup.baremetal.fini
+)
+
+
+set(TARGET_LLVMLIBC_ENTRYPOINTS
+  ${TARGET_LIBC_ENTRYPOINTS}
+)
diff --git a/libc/config/baremetal/x86_64/headers.txt b/libc/config/baremetal/x86_64/headers.txt
new file mode 100644
index 000000000..300addb7a
--- /dev/null
+++ b/libc/config/baremetal/x86_64/headers.txt
@@ -0,0 +1,26 @@
+set(TARGET_PUBLIC_HEADERS
+    libc.include.assert
+    libc.include.complex
+    libc.include.ctype
+    libc.include.errno
+    libc.include.features
+    libc.include.fenv
+    libc.include.float
+    libc.include.inttypes
+    libc.include.limits
+    libc.include.locale
+#    libc.include.math
+    libc.include.setjmp
+    libc.include.stdbit
+    libc.include.stdckdint
+#    libc.include.stdfix
+    libc.include.stdint
+    libc.include.stdio
+    libc.include.stdlib
+    libc.include.string
+    libc.include.strings
+    libc.include.sys_queue
+    libc.include.time
+    libc.include.uchar
+    libc.include.wchar
+)
diff --git a/libc/src/CMakeLists.txt b/libc/src/CMakeLists.txt
index a665253c4..262a1e780 100644
--- a/libc/src/CMakeLists.txt
+++ b/libc/src/CMakeLists.txt
@@ -7,9 +7,9 @@ add_subdirectory(errno)
 add_subdirectory(fenv)
 add_subdirectory(inttypes)
 add_subdirectory(link)
-add_subdirectory(math)
+# add_subdirectory(math)
 add_subdirectory(stdbit)
-add_subdirectory(stdfix)
+# add_subdirectory(stdfix)
 add_subdirectory(stdio)
 add_subdirectory(stdlib)
 add_subdirectory(string)
diff --git a/libc/src/stdlib/atof.cpp b/libc/src/stdlib/atof.cpp
deleted file mode 100644
index d0d8d211d..000000000
--- a/libc/src/stdlib/atof.cpp
+++ /dev/null
@@ -1,25 +0,0 @@
-//===-- Implementation of atof --------------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include "src/stdlib/atof.h"
-#include "src/__support/common.h"
-#include "src/__support/libc_errno.h"
-#include "src/__support/macros/config.h"
-#include "src/__support/str_to_float.h"
-
-namespace LIBC_NAMESPACE_DECL {
-
-LLVM_LIBC_FUNCTION(double, atof, (const char *str)) {
-  auto result = internal::strtofloatingpoint<double>(str);
-  if (result.has_error())
-    libc_errno = result.error;
-
-  return result.value;
-}
-
-} // namespace LIBC_NAMESPACE_DECL
diff --git a/libc/src/stdlib/strfromf.h b/libc/src/stdlib/strfromf.h
deleted file mode 100644
index e65e6b3a5..000000000
--- a/libc/src/stdlib/strfromf.h
+++ /dev/null
@@ -1,22 +0,0 @@
-//===-- Implementation header for strfromf ------------------------*- C++--===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#ifndef LLVM_LIBC_SRC_STDLIB_STRFROMF_H
-#define LLVM_LIBC_SRC_STDLIB_STRFROMF_H
-
-#include "src/__support/macros/config.h"
-#include <stddef.h>
-
-namespace LIBC_NAMESPACE_DECL {
-
-int strfromf(char *__restrict s, size_t n, const char *__restrict format,
-             float fp);
-
-} // namespace LIBC_NAMESPACE_DECL
-
-#endif // LLVM_LIBC_SRC_STDLIB_STRFROMF_H
diff --git a/libc/src/stdlib/strtod.cpp b/libc/src/stdlib/strtod.cpp
deleted file mode 100644
index deb2390c7..000000000
--- a/libc/src/stdlib/strtod.cpp
+++ /dev/null
@@ -1,29 +0,0 @@
-//===-- Implementation of strtod ------------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include "src/stdlib/strtod.h"
-#include "src/__support/common.h"
-#include "src/__support/libc_errno.h"
-#include "src/__support/macros/config.h"
-#include "src/__support/str_to_float.h"
-
-namespace LIBC_NAMESPACE_DECL {
-
-LLVM_LIBC_FUNCTION(double, strtod,
-                   (const char *__restrict str, char **__restrict str_end)) {
-  auto result = internal::strtofloatingpoint<double>(str);
-  if (result.has_error())
-    libc_errno = result.error;
-
-  if (str_end != nullptr)
-    *str_end = const_cast<char *>(str + result.parsed_len);
-
-  return result.value;
-}
-
-} // namespace LIBC_NAMESPACE_DECL
diff --git a/libc/src/stdlib/strtod_l.cpp b/libc/src/stdlib/strtod_l.cpp
deleted file mode 100644
index ad333b32d..000000000
--- a/libc/src/stdlib/strtod_l.cpp
+++ /dev/null
@@ -1,30 +0,0 @@
-//===-- Implementation of strtod_l ----------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include "src/stdlib/strtod_l.h"
-#include "src/__support/common.h"
-#include "src/__support/libc_errno.h"
-#include "src/__support/macros/config.h"
-#include "src/__support/str_to_float.h"
-
-namespace LIBC_NAMESPACE_DECL {
-
-LLVM_LIBC_FUNCTION(double, strtod_l,
-                   (const char *__restrict str, char **__restrict str_end,
-                    locale_t)) {
-  auto result = internal::strtofloatingpoint<double>(str);
-  if (result.has_error())
-    libc_errno = result.error;
-
-  if (str_end != nullptr)
-    *str_end = const_cast<char *>(str + result.parsed_len);
-
-  return result.value;
-}
-
-} // namespace LIBC_NAMESPACE_DECL
diff --git a/libc/src/stdlib/strtof.cpp b/libc/src/stdlib/strtof.cpp
deleted file mode 100644
index fc52dc85f..000000000
--- a/libc/src/stdlib/strtof.cpp
+++ /dev/null
@@ -1,29 +0,0 @@
-//===-- Implementation of strtof ------------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include "src/stdlib/strtof.h"
-#include "src/__support/common.h"
-#include "src/__support/libc_errno.h"
-#include "src/__support/macros/config.h"
-#include "src/__support/str_to_float.h"
-
-namespace LIBC_NAMESPACE_DECL {
-
-LLVM_LIBC_FUNCTION(float, strtof,
-                   (const char *__restrict str, char **__restrict str_end)) {
-  auto result = internal::strtofloatingpoint<float>(str);
-  if (result.has_error())
-    libc_errno = result.error;
-
-  if (str_end != nullptr)
-    *str_end = const_cast<char *>(str + result.parsed_len);
-
-  return result.value;
-}
-
-} // namespace LIBC_NAMESPACE_DECL
diff --git a/libc/src/stdlib/strtof_l.cpp b/libc/src/stdlib/strtof_l.cpp
deleted file mode 100644
index c6e03ff51..000000000
--- a/libc/src/stdlib/strtof_l.cpp
+++ /dev/null
@@ -1,30 +0,0 @@
-//===-- Implementation of strtof_l ----------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include "src/stdlib/strtof_l.h"
-#include "src/__support/common.h"
-#include "src/__support/libc_errno.h"
-#include "src/__support/macros/config.h"
-#include "src/__support/str_to_float.h"
-
-namespace LIBC_NAMESPACE_DECL {
-
-LLVM_LIBC_FUNCTION(float, strtof_l,
-                   (const char *__restrict str, char **__restrict str_end,
-                    locale_t)) {
-  auto result = internal::strtofloatingpoint<float>(str);
-  if (result.has_error())
-    libc_errno = result.error;
-
-  if (str_end != nullptr)
-    *str_end = const_cast<char *>(str + result.parsed_len);
-
-  return result.value;
-}
-
-} // namespace LIBC_NAMESPACE_DECL
